draft-ietf-capport-architecture.xml

```diff
@@ -393,24 +384,15 @@
            from the Captive Portal back-end.</t>
        </list>
       </t>
-      <t>
-        As an upgrade path, captive portals MAY continue to support methods
-        that work today, such as modification of port-80 HTTP responses to
-        redirect users to the portal.  Various user-equipment vendors probe
-        canary URLs to identify the state captivity [reference Mariko
-        Kobayashi's survey]. While doing so, Captive Portal Signals SHOULD also be sent,
-        to activate work-flows in supporting devices.
-        [TODO: give some thought to precise recommendations for backwards compatibility.]
-      </t>
      </section>
      <section anchor="section_signal" title="Captive Portal Signal">
        <t>
         User Equipment may send traffic outside the captive network prior to the Enforcement
         device granting it access. The Enforcement Device rightly blocks or resets these
-        requests. However, lacking a signal from the Enforcement Device, the User Equipment
-        can only guess at whether it is captive. Consequently, allowing the Enforcement Device
-        to signal to the User Equipment that there is a problem with its connectivity may
-        improve the user's experience.
+        requests. However, lacking a signal from the Enforcement Device or a visit
```

Comment:
Rather than "a visit to the API", can we use "communication with the API server" or  "interaction with the API server"